src/material-experimental/mdc-menu/menu-item.ts

```diff
@@ -19,6 +19,9 @@ import {MatMenuItem as BaseMatMenuItem} from '@angular/material/menu';
   inputs: ['disabled', 'disableRipple'],
   host: {
     '[attr.role]': 'role',
+    // The MatMenuItem parent class adds `mat-menu-item` to the CSS classlist, but this should
+    // not be added for this MDC equivalent menu item.
+    '[class.mat-menu-item]': 'false',
```

Comment:
AFAIK Ivy and ViewEngine have different behavior when it comes to resolving conflicting class bindings so I'm not sure how this will behave. Maybe we should just set the class manually in the constructor?

Reply:
I think there's a few approaches to doing this but none seem to be the ideal one. I don't particularly have a strong preference over any certain solution, and Jeremy and I just happened to land on this as the simplest. Based on what I see when running the devapp, this works fine

Reply:
The dev app runs in Ivy, it's ViewEngine that's inconsistent with how it handles conflicting bindings.

Reply:
You probably know more than I do with these conflicts, but I don't believe host bindings in the component metadata don't pass down at all to subclassed components. Do you know how to run the devapp in VE so I can test it out and confirm

Reply:
Hmm, good point that inheriting `host` didn't work on VE. In that case it should be fine. I don't know how we can run the dev app in VE.